layouts/default.html

```diff
@@ -21,6 +21,9 @@
                 <li><%= link_to "Tags", '/tags/' %></li>
                 <li><%= link_to "Atom", '/atom/' %></li>
                 <li><%= link_to "XML Sitemap", '/sitemap/' %></li>
+                <li><a href='mailto:orga[at]unfug(dot)org'>Mail us</a></li>
+                <li><a href='irc://irc.ghb.fh-furtwangen.de:6667/#unfug'>IRC</a></li>
```

Comment:
For those that don't support the protocol will have to check the source or hope their browser shows it on hover, please put it in clear text as well.